src/pytorch_lightning/strategies/deepspeed.py

```diff
@@ -396,7 +403,9 @@ def _set_node_environment_variables(self) -> None:
     def restore_checkpoint_after_setup(self) -> bool:
         return True
 
-    def _setup_model_and_optimizers(self, model: Module, optimizers: List[Optimizer]) -> Tuple[Module, List[Optimizer]]:
+    def _setup_model_and_optimizers(
+        self, model: Module, optimizers: List[Optimizer]
+    ) -> Tuple[DeepSpeedEngine, List[Optimizer]]:
```

Comment:
Hi @donlapark, this seems to break tests at runtime as `DeepSpeedEngine` is not available. https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING

Reply:
Thanks! Let's see if `"deepspeed.DeepSpeedEngine"` works.

Reply:
Alright, it works!